ansible/roles/riak_monitoring/templates/riak-restarter.sh.j2

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+# set the path so that service can be found
+PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games
+result=`service riak status`
```

Comment:
the conventional way to do that is to explicitly call `/usr/sbin/service` instead of using `PATH`